src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.test.tsx

```diff
@@ -22,7 +22,7 @@ describe('ContactDetailTab', () => {
       }
     `);
     expect(result.current.data.contact.name).toMatchInlineSnapshot(
-      `"Ship Star"`,
+      `"Elephant Film Stomach"`,
```

Comment:
@TheNoodleMoose I can switch to toMatchSnapshot this will create a snapshot file, instead of writing it inline.  I will make that change and let me know if you like that way better.